ecosystem/typescript/sdk/src/aptos_client.test.ts

```diff
@@ -8,7 +8,7 @@ import { AptosAccount } from "./aptos_account";
 import { TxnBuilderTypes, TransactionBuilderMultiEd25519, TransactionBuilderRemoteABI } from "./transaction_builder";
 import { TokenClient } from "./token_client";
 import { HexString } from "./hex_string";
-import { FAUCET_URL, NODE_URL } from "./utils/test_helper.test";
+import { FAUCET_URL as faucetUrl, NODE_URL as nodeUrl } from "./utils/test_helper.test";
```

Comment:
better to add the `!` in the source file than doing this kinda weird alias here imo:
```
export const NODE_URL = process.env.APTOS_NODE_URL!;
```